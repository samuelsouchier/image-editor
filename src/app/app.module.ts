import { NgModule } from '@angular/core';
import { BrowserModule } from '@angular/platform-browser';

import { AppComponent } from './app.component';
import { HeaderComponent } from './components/header/header.component';
import { EditorComponent } from './pages/editor/editor.component';

@NgModule({
  declarations: [
    AppComponent,
  ],
  imports: [
    BrowserModule,
    HeaderComponent,
    EditorComponent,
  ],
  bootstrap: [AppComponent]
})
export class AppModule { }
