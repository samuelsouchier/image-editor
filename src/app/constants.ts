export const BASE_IMAGE_HEIGHT = 300;
export const ZOOM_INCREMENT = 0.1;
