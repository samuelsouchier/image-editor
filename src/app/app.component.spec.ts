import { createComponentFactory, Spectator } from '@ngneat/spectator/jest';
import { AppComponent } from './app.component';

describe(AppComponent.name, () => {
  let spectator: Spectator<AppComponent>;
  const createComponent = createComponentFactory({
    component: AppComponent,
    shallow: true,
  });

  it('should match snapshot', () => {
    spectator = createComponent();

    expect(spectator.fixture).toMatchSnapshot();
  });
});
