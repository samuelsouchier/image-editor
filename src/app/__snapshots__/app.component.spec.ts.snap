// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppComponent should match snapshot 1`] = `
<app-root
  title={[Function String]}
>
  <app-header /><app-editor />
</app-root>
`;
